packages/cli/src/ui/utils/commandUtils.ts

```diff
@@ -44,23 +46,33 @@ export const copyToClipboard = async (text: string): Promise<void> => {
           ),
         );
       });
-      child.stdin.on('error', reject);
-      child.stdin.write(text);
-      child.stdin.end();
+      if (child.stdin) {
+        child.stdin.on('error', reject);
+        child.stdin.write(text);
+        child.stdin.end();
+      } else {
+        reject(new Error('Child process has no stdin stream to write to.'));
+      }
     });
 
+  // Configure stdio for Linux clipboard commands.
+  // - stdin: 'pipe' to write the text that needs to be copied.
+  // - stdout: 'inherit' since we don't need to capture the command's output on success.
+  // - stderr: 'pipe' to capture error messages (e.g., "command not found") for better error handling.
+  const linuxOptions: SpawnOptions = { stdio: ['pipe', 'inherit', 'pipe'] };
+
   switch (process.platform) {
     case 'win32':
       return run('clip', []);
     case 'darwin':
       return run('pbcopy', []);
```

Comment:
out of curiousity, is there a reason we shouldn't also be piping stderr for pbcopy. on mac. would imagine it might also send error output to stderr.